open-sphere-base/control-panels/src/main/java/io/opensphere/controlpanels/layers/util/FeatureTypeLabel.java

```diff
@@ -338,6 +338,12 @@ public void setIconByType(DataTypeInfo dti)
             MapVisualizationType type = dti.getMapVisualizationInfo().getVisualizationType();
             setIconByType(dti.getBasicVisualizationInfo().getTypeColor(), type);
         }
+        else if (dti != null && dti.getClass().getSimpleName().equals("CSVDataTypeInfo"))

```

Comment:
do an `instanceof` CSVDataTypeInfo

Reply:
I don't think Control Panels should know anything about CSV